getting-started-guides/on-prem-cluster/yarn-python.md

```diff
@@ -225,8 +249,8 @@ ${SPARK_HOME}/bin/spark-submit
  --py-files ${SPARK_PY_FILES}                                                   \
  ${SPARK_PYTHON_ENTRYPOINT}                                                     \
  --mainClass=${EXAMPLE_CLASS}                                                   \
- --dataPath=train::${DATA_PATH}/mortgage/csv/train/mortgage_train_merged.csv      \
- --dataPath=trans::${DATA_PATH}/mortgage/csv/test/mortgage_eval_merged.csv         \
+ --dataPath=train::${DATA_PATH}/mortgage/out/train/       \
+ --dataPath=trans::${DATA_PATH}/mortgage/out/eval/         \
  --format=csv                                                                   \
```

Comment:
Should the format be parquet?

Reply:
@shotai please help confirm.

Reply:
updated after profiling the whole process